lib/airbrake/rake/tasks.rb

```diff
@@ -1,67 +1,42 @@
+require 'airbrake-ruby'
+
 namespace :airbrake do
   desc 'Verify your gem installation by sending a test exception'
-  task test: :environment do
-    require 'pp'
-
-    begin
-      raise StandardError, 'Exception from the test Rake task'
-    rescue => ex
-      response = Airbrake.notify_sync(ex)
+  task test: (:environment if defined?(Rails)) do
+    if Airbrake[:default].is_a?(Airbrake::NilNotifier)
+      raise 'The Airbrake library is loaded but not configured'
```

Comment:
I can't use `Airbrake.notify` because it's a task which can quit earlier than we can send an exception (if this is what you meant).